html-css/week1/Homework 1/styles.css

```diff
@@ -0,0 +1,540 @@
+/*General*/
+@import url('https://fonts.googleapis.com/css2?family=Lato&family=Poppins:wght@300;400;500;600&display=swap');
+*{
+    padding: 0;
+    margin: 0;   
+}
+
+body{
+    font-family:'poppins',sans-serif;
+    background-color: #faf5fa;
+}
+html{
+    scroll-behavior: smooth;
+}
+p{
+    color: rgb(85, 85, 85);
+}
+/*Navigation desctop*/
+nav{
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 100%;           
+    height: 60px; 
+    background-color: white;
+}
+.logo{
+    background-color: white;
+    background-image:linear-gradient(to top,#080008 10%, #ef03fb 100%) ;
+    background-size: 100%;
+    -webkit-background-clip: text;
+    -moz-background-clip: text;
+    -webkit-text-fill-color: transparent;
+    -moz-text-fill-color:transparent;
+    display: flex;
+    align-items: center;
+    text-decoration: none;
+    font-size: 2rem;
+    
+}
+
+.nav-links{
+    display: flex;
+    gap:2rem ;
+    list-style: none;
+    font-size: 1.5rem;
+}
+a{
+    color: black;
+    text-decoration: none;
+    text-decoration-color: white;
+}
+a:hover{
+    color: rgb(96, 38, 84);
+   
+    text-decoration:underline;
+    text-decoration-color: rgb(94, 54, 95);
+    transition: all 400ms ease;
+}
+/*Section*/
+#home{
+    
+    margin: 0 10rem;

```

Comment:
you have 2 margin and also you used px and rem together.  remove second margin and merge it to the first one with rem.